### ISSUE TITLE
Release purl uri is not resolving

### ISSUE THREAD

**Issue by anitacaron** [glygen-glycan-data/GNOme#55]
I ran the OBO Dashboard yesterday and encountered an error while attempting to download the GNO. Checking the repository, I see that the assets were not uploaded in the last release.

I'll enjoy the moment to remember to update the [purl configuration](https://github.com/OBOFoundry/purl.obolibrary.org/blob/master/config/gno.yml) after a release. This is the only error on the OBO Dashboard. 🎉 

<img width="604" alt="Screenshot 2024-12-18 at 07 39 12" src="https://github.com/user-attachments/assets/c8a3991a-0840-46d1-ad89-cccd599a54f8" />

**Comment by edwardsnj** [comment]
@anitacaron Thanks for pointing this out. Not sure why I didn't notice the issue when you posted it. I've pushed a release for GNOme and updated the gno.yml file. Will this resolve the "failed to compute hashcode" issue too? Thanks

**Comment by anitacaron** [comment]
Yes, I'm running a new Dashboard update, and it got the latest gno 2025-01-28 version. 🎉

**Comment by edwardsnj** [comment]
Great, thanks!
